encrypted-media/resources/retrieve-persistent-usage-record.html

```diff
@@ -29,7 +29,7 @@
                                 sessionTypes: [ 'persistent-usage-record' ] },
             assertions = [ ];
 
-            config.messagehandler = messagehandler;
+            config.messagehandler = (new MessageHandler('org.w3.keysystem')).messagehandler;
```

Comment:
Looking further up, it looks like this file needs to be renamed. I think we discussed that in the other PR.